lib/apartment.rb

```diff
@@ -8,6 +8,7 @@
 
 require_relative 'apartment/log_subscriber'
 
+require_relative 'apartment/active_record/postgresql_adapter'
```

Comment:
This would force the loading of the pg adapter. Are other dbs no longer supported? This was my comment on https://github.com/rails-on-services/apartment/pull/143#discussion_r552701204

Reply:
Yes. I have not looked at how can we fix this and test it on a non postgresql db. I currently run only postgres so it's also hard for me to test these details on all dbs. I've been tempted on starting refactoring the gem to have one gem per adapter and a core, but that's discussion for another day.

Reply:
perhaps we can go the "easy" route:

```ruby
if defined?(PG)
  require_relative 'apartment/active_record/postgresql_adapter'
end
```

Perhaps in an initializer so that this happens after all gems are loaded?

Reply:
I think i have a clearer working solution. We only require the adapter once we eval the config. I'm requiring this patch after requiring the postgres adapter